.github/workflows/scheduled_release.yml

```diff
@@ -0,0 +1,10 @@
+on:
+  schedule:
+    # Every Monday at 14:00
```

Comment:
UTC?

Reply:
Probably, it doesn't really matter to me though.